plugins/utils/trimmer/copy.c

```diff
@@ -91,67 +94,43 @@ enum bt_component_status update_packet_context_field(FILE *err,
 	struct bt_ctf_field *field = NULL, *writer_field = NULL;
 	int nr_fields, i, int_ret;
 
+	BT_LOGI("Updating packet context field: name=%s", name);
 	packet_context = bt_ctf_packet_get_context(writer_packet);
-	if (!packet_context) {
-		fprintf(err, "[error] %s in %s:%d\n", __func__, __FILE__,
-				__LINE__);
-		goto error;
-	}
+	assert(packet_context);
```

Comment:
Yes in this function we know it exists, see the "goto end_no_notif" on iterator.c:438 if it does not exists